### ISSUE TITLE
Add "missing location" indicator in admin group listing 

### ISSUE THREAD

**Issue by tiptop96** [CAWSCIT/ca-meeting-finder-issue-tracker#37]
This will probably only be temporary so maybe we skip it. But to make it easier for an area (and us) to find meetings that have neither a physical location nor a virtual location we could give them some kind of indicator in the admin list view.

**Comment by KalobTaulien** [comment]
I agree, this is a good idea. Should be somewhat simple to implement in the ListView.
